sources/packages/forms/src/form-definitions/studentaccountactivity.json

```diff
@@ -530,6 +530,28 @@
             }
           ]
         },
+        {
+          "title": "Code PTD - Account Closed by Province",
+          "customClass": "collapsible",
+          "collapsible": true,
+          "key": "codePTDTitle",
+          "type": "panel",
+          "label": "Panel",
+          "collapsed": true,
+          "input": false,
+          "tableView": false,
+          "components": [
+            {
+              "html": "<p>A restriction has been placed on your file by StudentAid BC which disqualifies you from receiving further funding. For further information please contact StudentAid BC at 1-800-561-1818 (within North America) or 1-778-309-4621 (outside North America).</p>",
```

Comment:
The content has an extra "or" in the sentence; the AC should be either changed or the biz should be asked to adjust the AC.
If some action has already been taken to have it adjusted, please disregard this comment.

Reply:
I adjusted the AC for the "or".